core/src/main/kotlin/org/evomaster/core/problem/graphql/GraphQLActionBuilder.kt

```diff
@@ -79,27 +91,127 @@ object GraphQLActionBuilder {
 
 
         }
+
+
         for (elementIntable in table) {
+
             if (elementIntable.tableName == "Mutation" || elementIntable.tableName == "Query") {
-                handleOperation(actionCluster, elementIntable?.tableName, elementIntable?.tableField, elementIntable?.tableType)
+
+                handleOperation(actionCluster, elementIntable.tableField, elementIntable.tableName,
+                        elementIntable.tableType,
+                        elementIntable.kindOfTableType.toString(),
+                        elementIntable.kindOfTableField.toString(),
+                        table,
+                        elementIntable.tableName.toString())
+
             }
         }
 
     }
-
     private fun handleOperation(
             actionCluster: MutableMap<String, Action>,
-            tableName: String?,
-            tableField: String?,
-            tableType: String?
+            methodName: String?,
+            methodType: String?,
+            tableType: String,
+            kindOfTableType: String,
```

Comment:
what is the difference between `tableType` and `kindOfTableType`?  add some documentation

Reply:
Done